users/management/commands/genreferences.py

```diff
@@ -0,0 +1,22 @@
+from django.core.management.base import BaseCommand
+
+from drfx import settings
+from users.models import ServiceSubscription
+
+from utils import referencenumber
+
+# Caution: This does some destructive operations. Run only if you are sure.
+
+
+class Command(BaseCommand):
+    help = "Generates missing reference numbers for services that don't have them"
+
+    def handle(self, *args, **options):
+        subscriptions = ServiceSubscription.objects.all()
```

Comment:
Why not filter in the queryset already? 

subscriptions = ServiceSubscription.objects.filter(reference_number=None)

Reply:
I think there was some reason None didn't work with my test db. Throwaway script, won't fix.